projects/Flarum/programs/Flarum/examples/basic.nix

```
@@ -0,0 +1,5 @@
+{ ... }:
+
+{
+  programs.Flarum.enable = true;
```

Should be lower case

```suggestion
  programs.flarum.enable = true;
```

---

From the source file, Flarum has a capital ` F `.